cartographer_ros/cartographer_ros/node.cc

```diff
@@ -322,6 +327,18 @@ void Node::PublishLocalTrajectoryData(const ::ros::TimerEvent& timer_event) {
         pose_msg.pose = ToGeometryMsgPose(tracking_to_map);
         tracked_pose_publisher_.publish(pose_msg);
       }
+      if (node_options_.publish_tracked_pose_from_odom) {
+        if (trajectory_data.trajectory_options.provide_odom_frame) {
+            Rigid3d odom_to_tracking = tracking_to_local;
```

Comment:
Why is this being defined instead of using `tracking_to_local` directly? If you need to use it, I suggest using const ref
```suggestion
            const auto& odom_to_tracking = tracking_to_local;
```